travel_platform/travel_app/views.py

```diff
@@ -16,15 +25,28 @@ def index(request):
 
 def coord(request):
     """
-    View added for checking of interaction of map from landing page with
-    server.
-    :param request:
-    :return:
+    This view receives coordinates of location from the GET request, sends it
+    to the Open Maps API. Than returns name of the location received from API
+    to the frontend.
+
+    :param request: data received from frontend, containing longitude and
+      latitude
+    :return: name of location gotten from open maps APInav
+    to
     """
 
-    """TODO RomanPryima: make function sending coordinates and returning
-    to the frontend name of location received from map API"""
+    api_section = 'api.open.map'
+
+    api_url = get_setting(CONF_PATH, api_section, 'API_URL')
+    api_key = get_setting(CONF_PATH, api_section, 'API_KEY')
+    coordinate_lon = request.GET.get('lon')
+    coordinate_lat = request.GET.get('lat')
+    location = ','.join([coordinate_lat, coordinate_lon])
+    params = {'key': api_key, 'location': location}
 
-    response = request.GET.get('lon') + " " + request.GET.get('lat')
+    location_request = requests.get(api_url, params=params)
+    location_data = location_request.json()
 
+    response = location_data['results'][0]['locations'][0]['adminArea5']
```

Comment:
I didn't find easier method, because there is a mix with tuples and lists enclosed into each other. But it has stable immutable structure.